src/dialog/tests/unit/Dialog.ts

```diff
@@ -400,17 +400,6 @@ registerSuite('Dialog', {
 				onRequestClose.calledOnce,
 				'onRequestClose handler called when close button is clicked'
 			);
-
-			properties = {
-				closeable: false,
-				open: true,
-				onRequestClose
-			};
-			h.trigger(`.${css.close}`, 'onclick', stubEvent);
```

Comment:
Removed this as now `trigger` throws an error when a selector cannot be found, this part of the test didn't really prove anything anyway.